src/helpers/preview.ts

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import { compile } from 'handlebars';
+import child_process from 'child_process';
+
+export default function preview(id: string, params: any): string {
+  const tmpFile = path.join(
```

Comment:
We should not create a file and open it, the preview function should work on a production server not just in local, we can use this instead: https://www.npmjs.com/package/express-handlebars and have a route `/preview/:template` in which we can preview email from different templates.